.gitignore

```diff
@@ -9,9 +9,13 @@ __pycache__
 *.pyd
 *.bak
 *.c
+skimage/feature/_haar.cpp
 *.new
 *.md5
 *.old
+.pytest_cache
+temp.tif
+.ropeproject
```

Comment:
What's going on here?

Reply:
`.ropeproject`

 is some atom thing. I'll remove it if you don't want it.
`.pytest_cache` is for running tests on your own PC. gets created I guess.

`temp.tif` is left behind by some test.